### ISSUE TITLE
[Sepformer Inference] Inference code for custom pretrained model

### ISSUE THREAD

**Issue by windspirit95** [speechbrain/speechbrain#1173]
Hi,
I have successfully trained the Sepformer to separate 2 audio source. The result in test set also ok.
But when I try the inference code in Huggingface site, the result audios are speed perturbed (got slow effect).
```
from speechbrain.pretrained import SepformerSeparation as separator
import torch
import torchaudio
import os

def separate_file(model, path):
    batch, fs_file = torchaudio.load(path)
    print(batch)
    batch = batch.to(model.device)
    fs_model = model.hparams.sample_rate

    # resample the data if needed
    if fs_file != fs_model:
        print(
            "Resampling the audio from {} Hz to {} Hz".format(
                fs_file, fs_model
            )
        )
        tf = torchaudio.transforms.Resample(
            orig_freq=fs_file, new_freq=fs_model
        ).to(model.device)
        batch = batch.mean(dim=0, keepdim=True)
        batch = tf(batch)

    est_sources = model.separate_batch(batch)
    est_sources = est_sources / est_sources.max(dim=1, keepdim=True)[0]
    return est_sources

input_path = "denoised_dataset/test/1s_mix_0000004/mixture.wav"
model = separator.from_hparams(source="/home/user/speechbrain/recipes/WSJ0Mix/separation/results/denoised_sepformer/backup/ckpt_126", savedir='models_weights/sepformer-denoiser', run_opts={"device":"cuda"})
symlink_path = os.path.basename(input_path)
if os.path.exists(symlink_path):
    os.remove(symlink_path) # Remove the symlink
num_spks = est_sources.shape[2]
for i in range(num_spks):
    torchaudio.save("test_audios/source{}hat.wav".format(i + 1), est_sources[:, :, i].detach().cpu(), 8000)

```
I also saw that the class used for inference here (SepformerSeparation) is different with the model class in train code (Separation(sb.Brain)).
So could you take a look whether we need another inference code for custom pretrained model? Thanks.

**Comment by windspirit95** [comment]
Ah I got the issue here, the model is trained with sampling rate = 16000, and I saved audios with sampling rate=8000, that cause issue :)
